### PR TITLE
fix/request-url-build

### PR DESCRIPTION
**Pull request by yehor-podporinov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0-rc.7] - 2023-04-25
```
Comment (ArtSuslov) [inline CHANGELOG.md]: 04-27

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
...
+### Fixed
+- `@distributedlab/fetcher` - `buildRequestURL` helper
```
Comment (ArtSuslov) [inline CHANGELOG.md]: add a bit more information about fix

**Review by ArtSuslov — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (napalmpapalam) [reply CHANGELOG.md]: - `@distributedlab/fetcher` - Base URL cropping in the case when it has a path

File: CHANGELOG.md (hunk 1 above)
Comment (napalmpapalam) [inline CHANGELOG.md]: use `yarn apply version 0.2.0-rc.7` to bump version in all packages

Comment (yehor-podporinov) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (yehor-podporinov) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 1 above)
Comment (yehor-podporinov) [reply CHANGELOG.md]: done

**Review by napalmpapalam — APPROVED** [review]

**Pull request merged**